File: Swift/프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.swift
import Foundation

func solution(_ my_string:String) -> Int {
    let operation_list = my_string.split(separator: " ")
    
    var answer = Int(operation_list[0]) ?? 0
    for index in stride(from: 1, to:operation_list.count, by: 2) {
        let num = Int(operation_list[index + 1]) ?? 0
        
        if operation_list[index] == "+" {
            answer += num
        } else {
           answer -= num 
        }    
    }
    
    return answer
}

File: Python3/프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    operation_list = my_string.split(" ")
    
    num1, num2 = int(operation_list[0]), int(operation_list[2])
    
    answer = int(operation_list[0])
    for index in range(1, len(operation_list), 2):
        num = int(operation_list[index + 1])
        
        if operation_list[index] == "+":
            answer += num
        else:
            answer -= num
            
    return answer